CHANGELOG.md

```diff
@@ -7,228 +7,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Fixing errors in EditGasDisplay ([#11748](https://github.com/MetaMask/metamask-extension/pull/11748))
-- Allow max priority fee to be below 1, and only require it to be greater than 0 ([#11749](https://github.com/MetaMask/metamask-extension/pull/11749))
-- Allows users to set a max priority fee below suggested, just showing a warning in that case ([#11750](https://github.com/MetaMask/metamask-extension/pull/11750))
-- Rename effective gas price field to total gas fee ([#11754](https://github.com/MetaMask/metamask-extension/pull/11754))
-- Fix ipfs dependency vulernability ([#11745](https://github.com/MetaMask/metamask-extension/pull/11745))
-- Remove estimate detail above advanced gas controls in non-1559 network ([#11744](https://github.com/MetaMask/metamask-extension/pull/11744))
-- Fall back to aggregator name when icon is unavailable (Swaps Load View) ([#11718](https://github.com/MetaMask/metamask-extension/pull/11718))
-- EIP-1559 - Show minium native currency in banner when on testnets ([#11743](https://github.com/MetaMask/metamask-extension/pull/11743))
-- Fix a position of a tooltip icon ([#11739](https://github.com/MetaMask/metamask-extension/pull/11739))
-- Add gasPrice to be used on non-1559 networks for transaction details on confirm screen ([#11741](https://github.com/MetaMask/metamask-extension/pull/11741))
-- EIP-1559 - Provide better validation for gas price and gas limit ([#11736](https://github.com/MetaMask/metamask-extension/pull/11736))
-- EIP-1559 - Fix education modal typo ([#11740](https://github.com/MetaMask/metamask-extension/pull/11740))
-- Ensure that gas price in popover updates when api provided estimate updates ([#11727](https://github.com/MetaMask/metamask-extension/pull/11727))
-- EIP-1559 - Ensure transaction detail font-size and icon colors are consistent with Figma design ([#11735](https://github.com/MetaMask/metamask-extension/pull/11735))
-- EIP-1559 - Return null from GasTiming if on non-1559 network ([#11733](https://github.com/MetaMask/metamask-extension/pull/11733))
+### Added
+- [#12066](https://github.com/MetaMask/metamask-extension/pull/12066): Enable token conversion rates for primary currencies on some non-Mainnet networks
+- [#12110](https://github.com/MetaMask/metamask-extension/pull/12110): Add search tab in import tokens for custom networks
+  - Tab will be hidden when the `Use Token Detection` setting is disabled
+- [#11798](https://github.com/MetaMask/metamask-extension/pull/11798): Add fake token warning and replacing `Add Token` labels with `Import Tokens`
+- [#11755](https://github.com/MetaMask/metamask-extension/pull/11755): Add `Refresh list` and `import tokens` link to the main page
+  - Links will be hidden when the `Use Token Detection` setting is disabled
+- [#11851](https://github.com/MetaMask/metamask-extension/pull/11851): Add Token Detection Toggle to Security & Privacy (Settings)
```

Comment:
I think per @danjm's comment we'll need a qualifier under this too, any verbiage suggestions? cc: @Gudahtt

Reply:
Maybe something like "This feature is experimental and is off by default".

Reply:
How about: `This feature is experimental and is off by default, when enabled, it will add auto-detected tokens to a user's asset list`?

Reply:
that or some other description of what it does

Reply:
Sure, that sounds fine

Reply:
done